src/fu-engine.c

```diff
@@ -5040,11 +5040,23 @@ fu_engine_get_host_machine_id (FuEngine *self)
 static void
 fu_engine_add_security_attrs_tainted (FuEngine *self, FuSecurityAttrs *attrs)
 {
+	gboolean disabled_plugins = FALSE;
+	GPtrArray *blacklist = fu_config_get_blacklist_plugins (self->config);
 	FwupdSecurityAttr *attr = fwupd_security_attr_new ("org.fwupd.Hsi.Plugins");
```

Comment:
Ohh, this needs to be `g_autoptr()` -- my bad. Can you fix that up pls.

Reply:
sure, same problem elsewhere it seems too.